src/main/java/com/toofifty/easyblastfurnace/state/CoalBagState.java

```diff
@@ -19,6 +19,11 @@
     @Getter
     private int coal;
 
+    public void setMaxCoal(int quantity)
+    {
+        MAX_COAL = quantity;
```

Comment:
Probably best to make it an instance variable, just so it's obvious it's no longer a constant. Could instead use static constants to set the value:

```java
coalBag.setMaxCoal(CoalBagState.SMITHING_CAPE_MAX_COAL);

Reply:
Thanks for pointing this out, I've made it a normal private int now.